### ISSUE TITLE
Screenreader issue for <aria-labelledby> attribute

### ISSUE THREAD

**Issue by sswweber** [vaadin/vaadin-text-field#473]
The description of text fields cannot be recognized by the screenreader. The <aria-labelledby> attribute is filled with the id values of the label and the respective text field. 
example: `aria-labelledby="vaadin-text-field-label-0 vaadin-text-field-input-0"`.
Therefore the screenreader cannot create a valid connection between the two.

**Comment by web-padawan** [comment]
Moved the issue to the correct repo. The change was intentional, see https://github.com/vaadin/vaadin-text-field/commit/a11bc6a85fc056533dfecdaed926c7687998ccff

@platosha have you encountered any issues while working on it?

**Comment by web-padawan** [comment]
Closing in favor of https://github.com/vaadin/web-components/issues/161 which describes the same problem in more detail.
